lib/PuppeteerSharp.Tests/ElementHandleTests/BoxModelTests.cs

```diff
@@ -29,7 +29,7 @@ public async Task ShouldWork()
 
             // Step 2: Add div and position it absolutely inside frame.
             var frame = Page.FirstChildFrame();
-            var divHandle = (await frame.EvaluateFunctionHandleAsync(@"() => {
+            var divHandle = (ElementHandle)await frame.EvaluateFunctionHandleAsync(@"() => {
```

Comment:
no null check after an `as` implied it can only be that type. hence a cast is better, since if the assumption is incorrect you will get a useful cast exception instead of a useless null ref exception